### PR TITLE
Feature/ceph mgr exporter

### PR DESCRIPTION
**Pull request by technowhizz**
Add endpoints for ceph into the s-k-c defaults

### CODE REVIEW

File: releasenotes/notes/add-ceph-mgr-exporter-endpoints-f7d3c8b1131f6982.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
```
Comment (markgoddard) [inline releasenotes/notes/add-ceph-mgr-exporter-endpoints-f7d3c8b1131f6982.yaml:3]: It still requires you to set the flag to enable ceph monitoring in prometheus.

Comment (markgoddard) [reply releasenotes/notes/add-ceph-mgr-exporter-endpoints-f7d3c8b1131f6982.yaml:3]: Which is correct, but the release note should mention it.

**Review by markgoddard — CHANGES_REQUESTED** [review]
Could you update the monitoring docs too?

File: releasenotes/notes/add-ceph-mgr-exporter-endpoints-f7d3c8b1131f6982.yaml (hunk 1 above)
Comment (technowhizz) [reply releasenotes/notes/add-ceph-mgr-exporter-endpoints-f7d3c8b1131f6982.yaml:3]: Sorted :)

**Comment by technowhizz** [comment]
> Could you update the monitoring docs too?

Done!

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -77,6 +77,13 @@ prometheus_node_exporter_extra_volumes:
   - "textfile:/var/lib/node_exporter/textfile_collector"
 prometheus_node_exporter_cmdline_extras: "--collector.textfile.directory=/var/lib/node_exporter/textfile_collector"
 
+# Set Ceph manager exporter endpoints
+{% if groups['mgrs'] | length > 0 %}
+prometheus_ceph_mgr_exporter_endpoints:
+{% for host in groups['mgrs'] %}
+  - "{{ admin_oc_net_name | net_ip(host) }}:9283"
```
Comment (Alex-Welsh) [inline etc/kayobe/kolla/globals.yml:84]: Have you considered making the port number a variable? I don't know enough about it to actually know if it's a sensible idea.

Comment (technowhizz) [reply etc/kayobe/kolla/globals.yml:84]: Its the standard port for the ceph dashboard. Also I dont think its exists as a variable within KA (unlike something like `prometheus_node_exporter_port` for example).

Comment (MoteHue) [reply etc/kayobe/kolla/globals.yml:84]: This port is the default, and I don't think it is ever changed anywhere. The variable to change this port in KA for example was removed a while ago: https://github.com/openstack/kolla-ansible/commit/4e6fe7a6da4d5471cd0425c781f25f3e9567d3a0

**Review by markgoddard — APPROVED** [review]

**Pull request merged**